examples/crowd-funding/README.md

```diff
@@ -268,13 +268,7 @@ check that we have received 110 tokens, in addition to the
 
 ```gql,uri=http://localhost:8080/chains/$CHAIN_0/applications/$APP_ID_0
 query {
-    accounts {
-        entry(
-            key: "User:7136460f0c87ae46f966f898d494c4b40c4ae8c527f4d1c0b1fa0f7cff91d20f"
-        ) {
-            value
-        }
-    }
+    balance(owner: "User:7136460f0c87ae46f966f898d494c4b40c4ae8c527f4d1c0b1fa0f7cff91d20f")
```

Comment:
Afaict this README is still targeting the (non-native) fungible app, so this won't work.

Reply:
Oh nice!

Should we remove the (auto-generated?) `accounts` query in the fungible app?

Reply:
Good idea! Let me try

Reply:
Good job with the change. However, I'm having some afterthoughts on this: perhaps we got it backward and should instead implement the original GraphQL interface of `fungible` manually for `native-fungible`. Right now, the APIs are consistent but downgraded to a relatively naive version (which doesn't allow browsing the accounts for instance).

What do you think @Twey @afck?

Maybe we'd better leave the auto-generated API and add a TODO to support the same API in `native-fungible` and kill `balance`?

Reply:
Yes, I guess the service is less important to be compatible than the contract. I'm fine with leaving it as a TODO.

Alternatively, we could just support both kinds of query in `fungible`, and only the naive one in `native-fungible`?

Reply:
As discussed during co-working time, will add `accounts` support for native fungible in a following PR, and keep using that everywhere